templates/openstack/master/files/clouds.yaml

```diff
@@ -0,0 +1,16 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/openstack/clouds.conf"
+contents:
+  inline: |
+    {{with cloudProviderConfig .}}
+    [Global]
+    username={{.Cloud.AuthInfo.Username}}
```

Comment:
will this make the username and pw generally visible from a machineconfig?

if so, this seems like a problem since the ignition configs are not generally protected.

we need to determine a way to deliver secret data to nodes securely.